simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/StandardExports.java

```diff
@@ -56,8 +58,19 @@ public StandardExports() {
   public List<MetricFamilySamples> collect() {
     List<MetricFamilySamples> mfs = new ArrayList<MetricFamilySamples>();
 
-    mfs.add(new CounterMetricFamily("process_cpu_seconds_total", "Total user and system CPU time spent in seconds.",
-        osBean.getProcessCpuTime() / NANOSECONDS_PER_SECOND));
+    try {
+      Method method = osBean.getClass().getMethod("getProcessCpuTime", null);
+      if (!method.isAccessible()) {
```

Comment:
calling by reflection because there will be different implementing classes on IBM/Oracle JDK.
setAccessible because it's public *native*